code/modules/assembly/igniter.dm

```diff
@@ -24,6 +24,9 @@
 	var/turf/location = get_turf(loc)
 	if(location)
 		location.hotspot_expose(1000,1000)
+		for(var/T in location.contents)
+			var/atom/A = T
+			A.fire_act()

```

Comment:
This will burn (i.e. becomes a pile of ash after a while) every objects on the tile. :-1: